scripts/zephyr_module.py

```diff
@@ -417,6 +464,10 @@ def main():
                              of Zephyr modules and west projects.
                              If a module or project is also a git repository
                              the current SHA revision will also be written.""")
+    parser.add_argument('--meta-state-propagate', action='store_true',
+                        help="""Propagate state of modules and west projects
+                             to the Zephyr SHA and if west is used, to the
+                             manifest SHA""")
```

Comment:
"to the suffix of the Zephyr SHA" maybe?

Propagating something to a SHA sounds weird.

Reply:
It sure does sound strange, thanks for spotting.

Reply:
done.